openlibrary/templates/type/edition/view.html

```diff
@@ -168,19 +169,10 @@
     <td valign="top" width="100%">
     $:macros.databarView(page)
     <div class="navEdition sansserif">
-        $ work = page.works and page.works[0]
-        $if work:
-            <strong>$work.edition_count edition$("s" if work.edition_count != 1 else "")</strong> of
-            <span title="$_('Go to the main page for %s', page.title)"><a href="$work.url()">$truncate(work.title or "", 60)</a></span>
-        $else:
-            <strong>$:_('1 edition')</strong> record for of $page.title
-        $ authors_byline = get_authors_byline(page)
-        $if authors_byline:
-            $_("by") <span title="$_('Author')">$:authors_byline</span>
-        $else:
-            $:_("by an <em>unknown author</em>")
+        $ edition_count = work.edition_count if work else 1
```

Comment:
It shouldn't; it's the equivalent of saying:

```
if work:
    edition_count = work.edition_count
else:
    edition_count = 1
```

Although, it could be the case that `work` is defined by `work.edition_count` is not. So we could change this to `edition_count = work.edition_count if work and work.edition_count else 1`

Reply:
Whoops! Yeah, you're right, nevermind; not sure what I was thinking :P